src/style/app.module.css

```diff
@@ -169,8 +169,8 @@
   --modal-width: 670px;
   --modal-max-width: 680px;
   /* Hover style */
-  --hover-shadow: 0 1px 3px 0 rgba(168, 199, 250, 1),
-    0 4px 8px 3px rgba(60, 64, 67, 0.15);
+  --hover-shadow:
```

Comment:
Convert the files still using this CSS file to use `src/style/app-fixed.module.css`